src/Components/test/testassets/TestServer/Startup.cs

```diff
@@ -63,27 +64,35 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseAuthentication();
 
             // Mount the server-side Blazor app on /subdir
-            app.Map("/subdir", subdirApp =>
+            app.Map("/subdir", app =>
```

Comment:
This seemed like a general improvement, but I'm willing to back down if others hate it. It's not valid to reference the *outer* builder here, and this is a way to prevent it from happening (allowed as of C# 8).

Reply:
Overall I like it, because many times I've referenced the wrong builder in nested cases and wasted time figuring out why my middleware didn't do the right thing.